webpack.config.js

```diff
@@ -53,14 +54,18 @@ module.exports = {
 		]
 	},
 	plugins: [
-		new webpack.optimize.CommonsChunkPlugin(
-			"vendor", // chunkName
-			"bundle.vendor.js" // filename
-		),
-		new webpack.optimize.UglifyJsPlugin({
-			compress: {
-				warnings: false
-			}
-		}),
+		new webpack.optimize.DedupePlugin(),
+		new webpack.optimize.CommonsChunkPlugin("js/bundle.vendor.js"),
 	]
 };
+
+if (!isWatch) {
```

Comment:
You could use the node_env directly from here if you wanted to filter it from production and development build.

```
process.env.NODE_ENV
```